src/test/java/uk/gov/hmcts/reform/unspec/bpmn/ClaimantResponseTest.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.unspec.bpmn;
+
+import org.camunda.bpm.engine.externaltask.ExternalTask;
+import org.camunda.bpm.engine.externaltask.LockedExternalTask;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class ClaimantResponseTest extends BpmnBaseTest {
+
+    public static final String TOPIC_NAME = "processCaseEvent";
+
+    public ClaimantResponseTest() {
+        super("camunda/claimant_response.bpmn", "CLAIMANT_RESPONSE");
+    }
+
+    @Test
+    void caseEventTaskShouldFireCaseEventExternalTask_whenStarted() {
+        //assert process has started
+        assertFalse(processInstance.isEnded());
+
+        //assert topic names
+        assertThat(getTopics()).containsOnly(TOPIC_NAME);
+
+        //assert message start event
+        assertThat(getProcessDefinitionByMessage("CLAIMANT_RESPONSE")).isNotNull();
```

Comment:
what does isNotNull tell us here? Can we assert that correct actvityId is started or something?

Reply:
It was telling us that there is a process definition triggered by given message. But added checking if process id is correct.